tasks/buildimages.py

```diff
@@ -33,3 +33,39 @@ def update_test_infra_definitions(ctx: Context, commit_sha: str, go_mod_only: bo
     os.chdir("test/new-e2e")
     ctx.run(f"go get github.com/DataDog/test-infra-definitions@{commit_sha}")
     ctx.run("go mod tidy")
+
+
+@task(
+    help={
+        "old_build_image_tag": "The old build image tag",
+        "new_build_image_tag": "The new build image tag",
+        "old_go_version": "The old Go version",
+        "new_go_version": "The new Go version",
+        "test_version": "Flag to indicate if this is a test version",
+    }
+)
```

Comment:
```suggestion
@task(
    help={
        "old_build_image_tag": "The old build image tag",
        "new_build_image_tag": "The new build image tag",
        "old_go_version": "The old Go version",
        "new_go_version": "The new Go version",
        "test_version": "Flag to indicate if this is a test version",
    },
    autoprint=True
)
```

See the comment below about the `return`